src/Kodify/BlogBundle/Resources/views/Post/view.html.twig

```diff
@@ -9,5 +9,19 @@
         <div class="panel-body">
             {{ post.content }}
         </div>
+        <div>
+            <div class="panel-heading"><h4>Comments</h4></div>
+            {% if comments is empty %}
+                <div class="panel-body">There are no comments on this post, let's create some!!</div>
+            {% else %}
+                {% for comment in comments %}
+                    <div class="panel-body">
+                        <div class="pull-right">by: {{ comment.author.name }} on: {{ comment.createdAt|date("m/d/Y") }}</div>
+                        <div>{{ comment.content }}</div>
+                    </div>
+                {% endfor %}
+            {% endif %}
+            <a href="{{ path('create_comment', {id: post.id}) }}">Create comment</a>
```

Comment:
Should be fairly easy using some Javascript to render the new content into the same page, but I don't know how to combine it right now. I'll try to look into it.